src/main/scala/gemmini/LoopConv.scala

```diff
@@ -1334,6 +1346,9 @@ class LoopConv (block_size: Int, coreMaxAddrBits: Int, reservation_station_size:
           !has_first_layer_optimizations.B || config_max_pixels_per_row === 0.U,
           1.U, config_max_pixels_per_row)
 
+        loop_being_configured.a_ex_spad_id := cmd.bits.cmd.rs1(19, 18)
+        loop_being_configured.b_ex_spad_id := cmd.bits.cmd.rs1(17, 16) 
```

Comment:
What do the `a/b_ex_spad_id` options control?

Reply:
These would be helpful pointers:
https://github.com/ucb-bar/gemmini-rocc-tests/blob/bb4895319b4a7f7181a613a3853bc82887000ee0/include/gemmini.h#L789
https://github.com/ucb-bar/gemmini/blob/9089ce8f7639b61c43c46beb014cb555e22cfc5e/src/main/scala/gemmini/LoopMatmul.scala#L967

If the operands fit into scratchpad, it gets reused, so that we don't have to move in the same operands again. It would be done by specifying the double-banking id that the operand first moved in.

If the operands do not fit to spad, then spad_id would be just 0, and it would operate normally. (move in each operand for all tiled loops)